src/object/obj_rpc.h

```diff
@@ -294,6 +294,7 @@ CRT_RPC_DECLARE(obj_sync, DAOS_ISEQ_OBJ_SYNC, DAOS_OSEQ_OBJ_SYNC)
 	((uint64_t)		(om_max_eph)		CRT_VAR)	\
 	((uint32_t)		(om_version)		CRT_VAR)	\
 	((uint32_t)		(om_tgt_idx)		CRT_VAR)	\
+	((int32_t)		(om_clear_conts)	CRT_VAR)	\
```

Comment:
first of all, I would rather use an opcode instead of this flag. Also, we need another 4-bytes as padding to make the members be 64-bytes aligned.

Reply:
@wangdi1 Would like your opinion here - should I add different migrate opcodes for the different types of migration? (fail, reint, later add / migrate)

Reply:
well, migration is a generic module, so it should not tell if the migration is for rebuild or reintegrate. But on the other hand, this destroy hack is only for reintegration.  
So it does not matter if you add clear_conts or use different opc, here, since we will remove that finally anyway.

Probably just keep it as it is for the moment?